modules/satoshi/tf-makefile.template

```diff
@@ -0,0 +1,28 @@
+#
+# DO NOT OVERRIDE THIS FILE. AUTO-GENERATED FROM 'make satoshi/update-makefile'
+#
+export HELP_FILTER ?= asdf|jsonnet|opa|pluto|satoshi|k8s
+-include $(shell curl -sSL -o .build-harness "https://git.io/build-harness"; echo .build-harness)
+export BUILD_HARNESS_PATH ?= $(shell 'pwd')
+export BUILD_HARNESS_EXTENSIONS_PATH ?= $(BUILD_HARNESS_PATH)/build-harness-extensions
+
+
+## Init build-harness and build-harness-extensions
+.PHONY: bootstrap
+bootstrap:
+	git init
+	make init
+	if [ ! -d "./build-harness-extensions" ]; then git submodule add https://github.com/mintel/build-harness-extensions.git build-harness-extensions ; fi
+	git submodule update --init --recursive
+	exit 0
+
+## Install tools
+.PHONY: install
+install:
+	make asdf/install
+	exit 0
+
+## Clean .terraform directory
```

Comment:
This is probably ok, but a more ideal solution would be to have the `clean` option part of a terraform module, so the usage becomes

```
make terraform/clean
```

it's really just a way to namespace features (since there might be other `clean` options too)

Might be worth introducing a new `/modules/terraform` directory for this, but it could well conflict with the upstream one from `build-harness`, so would need testing

Reply:
Sounds good. Documentation states we should be fine as their terraform module doesn't have a clashing "clean" command. Going to add it now and test

Reply:
Tested it and it comes up with a few issues. I think I'll leave out clean for now and rely on the .gitignore to deal with it